### PR TITLE
Add Python Durable Functions templates

### LINKED ISSUE #929: Add Durable Functions 2.0 templates for Python

### PR DESCRIPTION
**Pull request by anthonychu**
Add Python Durable 1.x and 2.x. Add to only bundles for now.

@priyaananthasankar @scgbear Can one of you please check that they're okay? (should be able to copy paste into app and run)

// @cgillum @davidmrdavid @ConnorMcMahon @Hazhzeng 

Resolves #929

### CODE REVIEW

File: Build/PackageFiles/ExtensionBundleTemplates-1.x.nuspec [hunk 1]
```diff
@@ -108,6 +108,9 @@
     <file src="templates/DurableFunctionsActivity-JavaScript/function.json" target="Templates/DurableFunctionsActivity-JavaScript/function.json" />
     <file src="templates/DurableFunctionsActivity-JavaScript/index.js" target="templates/DurableFunctionsActivity-JavaScript/index.js" />
     <file src="templates/DurableFunctionsActivity-JavaScript/metadata.json" target="Templates/DurableFunctionsActivity-JavaScript/metadata.json" />
+    <file src="templates/DurableFunctionsActivity-Python-1.x/function.json" target="Templates/DurableFunctionsActivity-Python-1.x/function.json" />
```
Comment (soninaren) [inline Build/PackageFiles/ExtensionBundleTemplates-1.x.nuspec]: the target should be `Templates/DurableFunctionsActivity-Python/function.json`

File: Build/PackageFiles/ExtensionBundleTemplates-2.x.nuspec [hunk 2]
```diff
@@ -108,6 +108,9 @@
     <file src="templates/DurableFunctionsActivity-JavaScript/function.json" target="Templates/DurableFunctionsActivity-JavaScript/function.json" />
     <file src="templates/DurableFunctionsActivity-JavaScript/index.js" target="templates/DurableFunctionsActivity-JavaScript/index.js" />
     <file src="templates/DurableFunctionsActivity-JavaScript/metadata.json" target="Templates/DurableFunctionsActivity-JavaScript/metadata.json" />
+    <file src="templates/DurableFunctionsActivity-Python-2.x/function.json" target="Templates/DurableFunctionsActivity-Python-2.x/function.json" />
```
Comment (soninaren) [inline Build/PackageFiles/ExtensionBundleTemplates-2.x.nuspec]: same as above target should be `Templates/DurableFunctionsActivity-Python/function.json`

File: Functions.Templates/Templates/DurableFunctionsOrchestrator-Python-2.x/metadata.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  "category": [
+    "$temp_category_core",
+    "$temp_category_durableFunctions"
+  ],
+  "categoryStyle": "other",
+  "enabledInTryMode": false,
+  "userPrompt": [],
+  "extensions": [
+    {
+      "id": "Microsoft.Azure.WebJobs.Extensions.DurableTask", "version": "2.2.0"
```
[the remark this answers is not in the document]
Comment (ConnorMcMahon) [reply Functions.Templates/Templates/DurableFunctionsOrchestrator-Python-2.x/metadata.json]: We will be releasing 2.2.0 in the next few days hopefully. This PR will need to wait to merge until that happens.

[the remark this answers is not in the document]
Comment (anthonychu) [reply Functions.Templates/Templates/DurableFunctionsOrchestrator-Python-2.x/metadata.json]: Thanks. Will change it and submit another PR when `2.2.0` is released.

File: Build/PackageFiles/ExtensionBundleTemplates-1.x.nuspec (hunk 1 above)
Comment (anthonychu) [reply Build/PackageFiles/ExtensionBundleTemplates-1.x.nuspec]: Good catch thank.

File: Build/PackageFiles/ExtensionBundleTemplates-2.x.nuspec (hunk 2 above)
Comment (anthonychu) [reply Build/PackageFiles/ExtensionBundleTemplates-2.x.nuspec]: Fixed this as well. I see the Durable C# ones have `-2.x`. I'll remove them too.

File: Functions.Templates/Templates/DurableFunctionsOrchestrator-Python-2.x/metadata.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (soninaren) [reply Functions.Templates/Templates/DurableFunctionsOrchestrator-Python-2.x/metadata.json]: Can we split the PR in two? each with templates from different major version of durable extension

**Pull request merged**